openlibrary/templates/site/body.html

```diff
@@ -22,8 +20,5 @@
       $for flash in get_flash_messages():
         <div class="$flash.type"><span>$flash.message</span></div>
       </div>
-      <div class="page-banner page-banner-body">
```

Comment:
Let's leave this element, but comment it out. That way we'll be able to easily bring it back. Copy the `announcement` inside here as well, so we have a reference of the `link-track`s we use.

Reply:
better yet, let's only render the component if there's an announcement and just remove the announcement.

Reply:
e.g.
```
$if announcement:
  <div class="page-banner page-banner-body">$:announcement</div>
```

Reply:
In the future we could potentially annotate announcements with date ranges or simply pull from our blog (as is done in the homepage) automatically for some fixed period of time (this comment not actionable on this issues, but may be used to create a new low-priority issue)

Reply:
Drats! No, I changed my mind; I want to keep the announcement around because it contains like the `ol-link` stuff. Maybe instead of commenting, creating a new variable called `show_banner` and set it to false. Then copy the `announcement` value into the editor, and wrap it in an `$if` clause.

Reply:
I named the variable _show_announcement_ to avoid confusion with the existing _show_banners_ variable